packages/examples/cvat/exchange-oracle/src/crons/process_recording_oracle_webhooks.py

```diff
@@ -83,29 +81,22 @@ def handle_recording_oracle_event(webhook: Webhook, *, db_session: Session, logg
                 for project in projects_chunk:
                     if project.status != ProjectStatuses.validation:
                         logger.error(
-                            "Unexpected event {} received for a project in the {} status, "
-                            "ignoring (escrow_address={}, project={})".format(
-                                webhook.event_type,
-                                project.status,
-                                webhook.escrow_address,
-                                project.cvat_id,
-                            )
+                            f"Unexpected event {webhook.event_type} received for a project in the {project.status} status, "
+                            f"ignoring (escrow_address={webhook.escrow_address}, project={project.cvat_id})"
                         )
                         return
 
                     new_status = ProjectStatuses.recorded
                     logger.info(
-                        "Changing project status to {} (escrow_address={}, project={})".format(
-                            new_status, webhook.escrow_address, project.cvat_id
-                        )
+                        f"Changing project status to {new_status} (escrow_address={webhook.escrow_address}, project={project.cvat_id})"
```

Comment:
It seems like the fixes actually introduce too long lines. I'd also say that there are cases where f-strings are harder to read and/or maintain (e.g. with long variable names or field access in substitutions, with many substitutions).

Reply:
Ah, I see. Ruff `UP032` seem to ignore the line-length requirement since https://github.com/astral-sh/ruff/pull/10263 was introduced.

From my experience, f-string are almost always more readable than stings with `.format()`, when used correctly, 

Yet I tend to stick with `.format()` here just to preserve line-length compliance without changing everything by hand.

Reply:
Disabled `UP032` for now. Hope `ruff` will get a flag to restore previous behaviour.